smpplib/client.py

```diff
@@ -63,10 +63,20 @@ class Client(object):
     _ssl_context = None
     sequence_generator = None
 
-    def __init__(self, host, port, timeout=5, sequence_generator=None, logger_name=None, ssl_context=None):
+    def __init__(
+        self,
+        host,
+        port,
+        timeout=5,
+        sequence_generator=None,
+        logger_name=None,
+        ssl_context=None,
+        allow_unknown_opt_params=False
```

Comment:
stylish, but could you please leave a coma here?
```suggestion
        allow_unknown_opt_params=False,
```

Reply:
Also, I'm thinking, that we could do `None` here and, if default value is used, set it to `False`  and raise a deprecation warning.